### PR TITLE
db_central_columns: new column collapsing issue

### LINKED ISSUE #12904: Collapsing issue @ db_central_columns
Maybe it should not collapse in case there are no columns defined.

https://github.com/phpmyadmin/phpmyadmin/issues/12834#issuecomment-273515440

### PR DESCRIPTION
**Pull request by poush**
Fixes: #12904 
Signed-off-by: Piyush Agrawal <poush12@gmail.com>

Before submitting pull request, please check that every commit:

- [X] Has proper Signed-Off-By
- [x] Has commit message which describes it
- [x] Is needed on it's own, if you have just minor fixes to previous commits, you can squash them
- [x] Any new functionality is covered by tests

### CODE REVIEW

**Comment by nijel** [comment]
When touching the code, can you please also remove the inline CSS, see https://github.com/phpmyadmin/phpmyadmin/issues/12262

**Comment by poush** [comment]
Done!

File: libraries/central_columns.lib.php [hunk 1]
```diff
@@ -1252,16 +1252,18 @@ function PMA_handleColumnExtra(&$columns_list)
...
  *                list
  */
-function PMA_getHTMLforAddNewColumn($db)
+function PMA_getHTMLforAddNewColumn($db, $total_rows)
 {
     $addNewColumn = '<div id="add_col_div" class="topmargin"><a href="#">'
         . '<span>+</span> ' . __('Add new column') . '</a>'
-        . '<form id="add_new" style="min-width:100%;display:none" '
+        . '<form id="add_new" class="new_central_col '
+        . ($total_rows != 0 ? 'hidden"' : '"')
```
Comment (nijel) [inline libraries/central_columns.lib.php]: Better to use existing class `hide` than introduce new one with same function.

**Comment by poush** [comment]
@nijel Actually I had "bootstrap" classes in my mind therefore searched for "hidden" class. Updated again!

**Comment by nijel** [comment]
Thanks, merged as b575f7ef1b6477f5240da31053d30ef75eb5ab49 (to include it in 4.7 maintenance branch).

**Pull request closed without merging**